src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadAction.java

```diff
@@ -46,32 +45,26 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         CourseRoster roster = new CourseRoster(logic.getStudentsForCourse(courseId),
                                                logic.getInstructorsForCourse(courseId));
         
-        data = new InstructorFeedbackResponseCommentsLoadPageData(account);
-        data.feedbackResultBundles = getFeedbackResultBundles(courseId, fsname, roster);
-        data.instructorEmail = instructor.email;
-        data.currentInstructor = instructor;
-        data.roster = roster;
-        data.feedbackSessionIndex = fsindex;
-        data.numberOfPendingComments = logic.getCommentsForSendingState(courseId, CommentSendingState.PENDING).size() 
+        int numberOfPendingComments = logic.getCommentsForSendingState(courseId, CommentSendingState.PENDING).size() 
                 + logic.getFeedbackResponseCommentsForSendingState(courseId, CommentSendingState.PENDING).size();
+        FeedbackSessionResultsBundle bundle = getFeedbackResultBundle(courseId, fsname, roster);
```

Comment:
You mean, why I changed it from returning a map of bundles to just a single one?

If you look at the old `getFeedbackResultBundles()`, you can see that it gets a single bundle from Logic, then proceeds to put it into the map right before returning that map.
So there's only a single bundle in that map and that fact doesn't change anywhere else in the old code.

`CommentsLoad` happens when you click on a feedback session heading on the comments page, which loads that specific feedback session which is connected to a specific bundle, so it makes sense for it to handle only a single bundle.

They might have had the idea to have it retrieve a list of bundles and load all of them at the same time, but it seems unimplemented and there aren't any to-dos, so I assume its current purpose is the intended purpose.